go/routers/app_multiplexer/tests/test_views.py

```diff
@@ -30,7 +30,11 @@ def test_show_stopped(self):
         router = response.context[0].get('router')
         self.assertEqual(router.name, u"myrouter")
         self.assertContains(response, rtr_helper.get_view_url('start'))
-        self.assertNotContains(response, rtr_helper.get_view_url('stop'))
+        self.assertContains(
+            response,
+            '<button class="btn btn-danger action" data-action="stop" ' +
+            ' data-url="%s" disabled>Deactivate</button>'
+            % rtr_helper.get_view_url('stop'), html=True)
```

Comment:
I think we usually just will do something like `self.assertContains(response, rtr_helper.get_view_url('stop'))` and `self.assertContains(response, '>Deactivate<')`, but I can't think of a reason for why testing these other things is necessarily bad. `disabled` is important here, so we definitely need that. Does seem a bit weird to be testing `button`, `class` and `data-action` though, since we are testing things hardcoded in the template, unless we want tests to fail if any of those change. Is that the case?